pkg/actions/nodegroup/scale.go

```diff
@@ -66,19 +67,19 @@ func (m *Manager) scaleUnmanagedNodeGroup(ng *api.NodeGroupBase, stackInfo manag
 	}
 
 	if ng.MaxSize != nil {
-		input.MaxSize = aws.Int64(int64(*ng.MaxSize))
+		input.MaxSize = aws.Int32(int32(*ng.MaxSize))
 	}
 
 	if ng.MinSize != nil {
-		input.MinSize = aws.Int64(int64(*ng.MinSize))
+		input.MinSize = aws.Int32(int32(*ng.MinSize))
 	}
 
 	if ng.DesiredCapacity != nil {
-		input.DesiredCapacity = aws.Int64(int64(*ng.DesiredCapacity))
+		input.DesiredCapacity = aws.Int32(int32(*ng.DesiredCapacity))
 	}
-	out, err := m.ctl.Provider.ASG().UpdateAutoScalingGroup(input)
+
+	_, err := m.ctl.Provider.ASG().UpdateAutoScalingGroup(ctx, input)
 	if err != nil {
-		logger.Debug("ASG update output: %s", out.String())
```

Comment:
We don't need this output anymore? 🤔

Reply:
`out.String()` is no longer a thing, so I wasn't really sure if it made sense to just print th estruct. I'm not really sure, but given its a debug and not info output i think its okay to just remove :shrug: wdyt?

Reply:
nah